pkgs/top-level/all-packages.nix

```diff
@@ -142,6 +142,8 @@ in
 
   ankisyncd = callPackage ../servers/ankisyncd { };
 
+  fiche = callPackage ../servers/fiche { };
```

Comment:
It would be great if you could sort this into a fitting category and alphabetically there.